src/View/Helper/BreadcrumbsHelper.php

```diff
@@ -0,0 +1,295 @@
+     * @param string $title Title of the crumb
+     * @param string|array|null $url URL of the crumb. Either a string, an array of route params to pass to
+     * Url::build() or null / empty if the crumb does not have a link
+     * @param array $options Array of options. These options will be used as attributes HTML attribute the crumb will
+     * be rendered in (a <li> tag by default). It accepts two special keys:
+     * - *innerAttrs*: An array that allows you to define attributes for the inner element of the crumb (by default, to
+     * the link)
+     * - *templateVars*: Specific template vars in case you override the templates provided
+     * @return $this
+     * @throws LogicException In case the matching crumb can not be found
+     */
+    public function insertAfter($matchingTitle, $title, $url = null, array $options = [])
+    {
+        $key = $this->findCrumb($matchingTitle);
+
+        if ($key !== null) {
+            return $this->insertAt($key + 1, $title, $url, $options);
+        }
+        throw new LogicException(sprintf("No crumb matching '%s' could be found.", $matchingTitle));
+    }
+
+    /**
+     * Returns the crumbs list
+     *
+     * @return array
+     */
+    public function getCrumbs()
+    {
+        return $this->crumbs;
+    }
+
+    /**
+     * Renders the breadcrumbs trail
+     *
+     * @param array $attributes Array of attributes applied to the `wrapper` template. Accepts the `templateVars` key to
+     * allow the insertion of custom template variable in the template.
+     * @param array $separator Array of attributes for the `separator` template.
+     * Possible properties are :
+     * - *separator* The string to be displayed as a separator
+     * - *templateVars* Allows the insertion of custom template variable in the template
+     * - *innerAttrs* To provide attributes in case your separator is divided in two elements
+     * All other properties will be converted as HTML attributes and will replace the *attrs* key in the template
+     * If you use the default for this option (empty), it will not render a separator.
+     * @return string The breadcrumbs trail
+     */
+    public function render(array $attributes = [], array $separator = [])
+    {
+        $crumbs = $this->crumbs;
+        $crumbsCount = count($crumbs);
+        $templater = $this->templater();
+        $separatorString = '';
+
+        if ($separator) {
+            if (isset($separator['innerAttrs'])) {
+                $separator['innerAttrs'] = $templater->formatAttributes($separator['innerAttrs']);
+            }
+
+            $separator['attrs'] = $templater->formatAttributes(
+                $separator,
+                ['innerAttrs', 'separator']
+            );
+
+            $separatorString = $this->formatTemplate('separator', $separator);
+        }
+
+        $crumbTrail = '';
+        foreach ($crumbs as $key => $crumb) {
```

Comment:
May be this foreach could even be extracted as protected method. Could make subclassing/extending easier. Not a big deal though.